site/src/sphinx/index.rst

```diff
@@ -45,15 +45,20 @@ Integration with gRPC and Thrift
 - Your gRPC or Thrift service implementation runs on top of Armeria without any modification.
 - Works with the official gRPC or Thrift-over-HTTP client
 - Works with the code generated by the official Protobuf or Thrift IDL compiler
-- Automatically upgrades to HTTP/2 when used with the Armeria client
+- Supports various protocol combinations, such as:
+
+  - gRPC-Web
```

Comment:
I guess gRPC-web is less commonly used so could go below in order. While a great, unique feature, I hope users of normal gRPC also feel like like switching to armeria :)

Reply:
Reordered. Thanks for a feed back!